modules/subscriptions.php

```diff
@@ -635,7 +635,7 @@ function widget( $args, $instance ) {
 		} else { ?>
 			<form action="#" method="post" accept-charset="utf-8" id="subscribe-blog-<?php echo $widget_id; ?>">
 				<?php
-				if ( ! isset ( $_GET['subscribe'] ) ) {
+				if ( ! isset ( $_GET['subscribe'] ) OR $_GET['subscribe'] != 'success' ) {
```

Comment:
Just a couple nitpicky code style requests: 

WP coding styles calls for <a href="https://make.wordpress.org/core/handbook/coding-standards/php/#yoda-conditions">yoda style conditions</a>.   We also typically use pipes for our logical operators.